### ISSUE TITLE
some class Iou is nan

### ISSUE THREAD

**Issue by zibuyu2018** [zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow#42]
![2018-04-12 18-12-07](https://user-images.githubusercontent.com/35693781/38672434-bdcb02b2-3e80-11e8-876d-9f290814edf1.png)
![2018-04-12 19-07-06](https://user-images.githubusercontent.com/35693781/38673735-bb83493e-3e84-11e8-8a8d-440cc1e2c132.png)

I downloaded the aug dataset and converted the mat format label to a png format .Then I trained with your train.txt file. I adjust the picture input_height and input_width  to 270*270 because of GPU(1080), after 20000 training loss is 1.18. I test the model with Test.txt but found Iou is very low  and some class value is nan. And every predict result only have two classes (tow colors) and some is wrong result.How can I solve it ?   Thank you very much.
.

**Comment by zibuyu2018** [comment]
I use deeplab_resnet.ckpt as pre-trained model.

**Comment by StevenVerne** [comment]
if TP+FP+FN is zero, then iou is nan

**Comment by zhengyang-wang** [comment]
The _visual.png files are for visualization. For evaluation, please use the prediction files which are not for visualization.

**Comment by zibuyu2018** [comment]
I solve this problem by delete each picture's colormap,thank you.

**Comment by daicoolb** [comment]
@zibuyu2018 , what do you mean by deleting each picture's colormap ?
